events/friends_of_kenilworth_gardens.py

```diff
@@ -60,6 +64,7 @@ def handle_simple(self, event, keys):
         elif len(keys) == 2:
             return event[keys[0]][keys[1]]
         else:
+            logger.error(f"Exception as its more than 2 levels : {e}", exc_info=True)
             raise Exception("Can't handle more than 2 levels...")
```

Comment:
I never noticed this `raise` on the line below, but we don't want that as that'll throw a break into our script instead of letting it get logged and then pass. Also, what does it mean when it says more than 2 levels? Could we be more explicit?

Reply:
One other thing down on line 127 where we have `html_doc = requests.get(url).content`. That GET should have some error handling around it. Something like

```
try:
    r = requests.get(url)
except Exception as e:
    logger.error(f"Exception occurred making GET request to {url}:  \
                            {e}", exc_info=True)
```